traffic_control/locale/fi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,331 @@
+msgstr "Vaijeri"
+
+#: traffic_control/models.py:14
+msgid "Bridge"
+msgstr "Silta"
+
+#: traffic_control/models.py:15 traffic_control/models.py:23
+msgid "Other"
+msgstr "Muu"
+
+#: traffic_control/models.py:19
+msgid "Active"
+msgstr "Aktiivinen"
+
+#: traffic_control/models.py:20
+msgid "Covered"
+msgstr "Huputettu"
+
+#: traffic_control/models.py:21
+msgid "Fallen"
+msgstr "Kaatunut"
+
+#: traffic_control/models.py:22
+msgid "Missing"
+msgstr "Kadonnut"
+
+#: traffic_control/models.py:27
+msgid "Small"
+msgstr "Pieni"
+
+#: traffic_control/models.py:28
+msgid "Medium"
+msgstr "Normaali"
+
+#: traffic_control/models.py:29
+msgid "Large"
+msgstr "Suuri"
+
+#: traffic_control/models.py:33
+msgid "Convex"
+msgstr "Kupera"
+
+#: traffic_control/models.py:34
+msgid "Direct"
+msgstr "Suora"
+
+#: traffic_control/models.py:38
+msgid "r1"
+msgstr "r1"
+
+#: traffic_control/models.py:39
+msgid "r2"
+msgstr "r2"
+
+#: traffic_control/models.py:40
+msgid "r3"
+msgstr "r3"
+
+#: traffic_control/models.py:44
+msgid "Blue"
+msgstr "Sininen"
+
+#: traffic_control/models.py:45
+msgid "Yellow"
+msgstr "Keltainen"
+
+#: traffic_control/models.py:49
+msgid "Right side"
+msgstr "Oikea puoli"
+
+#: traffic_control/models.py:50
+msgid "Left side"
+msgstr "Vasen puoli"
+
+#: traffic_control/models.py:51
+msgid "Above"
+msgstr "Kaistan yläpuolella"
+
+#: traffic_control/models.py:52
+msgid "Middle"
+msgstr "Keskisaareke tai liikenteenjakaja"
+
+#: traffic_control/models.py:53
+msgid "Vertical"
+msgstr "Pitkittäin ajosuuntaan nähden"
+
+#: traffic_control/models.py:54
+msgid "Outside"
+msgstr "Tie tai katuverkon ulkopuolella"
+
+#: traffic_control/models.py:58
+msgid "Very bad"
+msgstr "Erittäin huono"
+
+#: traffic_control/models.py:59
+msgid "Bad"
+msgstr "Huono"
+
+#: traffic_control/models.py:60
+msgid "Average"
+msgstr "Tyydyttävä"
+
+#: traffic_control/models.py:61
+msgid "Good"
+msgstr "Hyvä"
+
+#: traffic_control/models.py:62
+msgid "Very good"
+msgstr "Erittäin hyvä"
+
+#: traffic_control/models.py:69
+msgid "Code"
+msgstr "Merkkikoodi"
+
+#: traffic_control/models.py:71 traffic_control/models.py:89
+msgid "Description"
+msgstr "Kuvaus"
+
+#: traffic_control/models.py:76
+msgid "Traffic Sign Code"
+msgstr "Merkkikoodi"
+
+#: traffic_control/models.py:77
+msgid "Traffic Sign Codes"
+msgstr "Merkkikoodit"
+
+#: traffic_control/models.py:87
+msgid "Status"
+msgstr "Tila"
+
+#: traffic_control/models.py:94
+msgid "Lifecycle"
+msgstr "Elinkaari"
+
+#: traffic_control/models.py:95
+msgid "Lifecycles"
+msgstr "Elinkaaret"
+
+#: traffic_control/models.py:105 traffic_control/models.py:200
+msgid "Location (2D)"
+msgstr "Sijainti (2D)"
+
+#: traffic_control/models.py:107 traffic_control/models.py:202
+msgid "Height"
+msgstr "Korkeus"
+
+#: traffic_control/models.py:109 traffic_control/models.py:204
+msgid "Direction"
+msgstr "Suunta"
+
+#: traffic_control/models.py:112 traffic_control/models.py:207
+msgid "Traffic sign code value"
+msgstr "Liikennemerkin arvo"
```

Comment:
"Traffic sign code" is translated as "Merkkikoodi" on L163. The translation should probably match so that same term is translated the same way for each occurance.

Reply:
Let's update it to "Merkkikoodin arvo".